doc/users/next_whats_new/alpha_array.rst

```diff
@@ -0,0 +1,31 @@
+Transparency (alpha) can be set as an array in collections
+----------------------------------------------------------
+Previously, the alpha value controlling tranparency in collections could be
+specified only as a scalar which was applied to all elements in the collection.
```

Comment:
Reads a bit longer than it needs to, I think.
```suggestion
specified only as a scalar applied to all elements in the collection.
```

Reply:
@QuLogic you have a good eye!  Fixed.